src/fswAlgorithms/transDetermination/ephemDifferenceWithUncertainty/ephemDifferenceWithUncertainty.h

```diff
@@ -42,12 +42,21 @@ class EphemDifferenceWithUncertainty: public SysModel {
     void UpdateState(uint64_t CurrentSimNanos);
     void Reset(uint64_t CurrentSimNanos);
 
+    void setCovarianceBase(const Eigen::MatrixXd stateCovariance);
+    const Eigen::MatrixXd getCovarianceBase() const;
```

Comment:
:small_red_triangle_down: We can make the return type on both of these getters non-const. The const has no effect on the return value.

Reply:
Done.